accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/aceservicejmx/impl/AceServiceMBeanImpl.java

```diff
@@ -53,41 +57,126 @@ public AceServiceMBeanImpl() throws NotCompliantMBeanException {
     @Reference
     Dumpservice dumpservice;
 
+    @Reference
+    SlingRepository repository;
+
     @Override
     public String apply() {
-        return aceService.execute().toString();
+        Session session = null;
+        try {
+            session = repository.loginService(Constants.USER_AC_SERVICE, null);
+            return aceService.execute(session).toString();
+        } catch (RepositoryException e) {
```

Comment:
I would rather rely on the exception handling of the code calling the mbean. So just throwing here is fine.